tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -254,12 +255,17 @@ func lastFlag(args []string) ([]string, error) {
     }
 
     if len(activations) == 0 {
-      whiskErr := whisk.MakeWskError(errors.New("No activations found"), whisk.EXITCODE_ERR_GENERAL, whisk.DISPLAY_MSG, whisk.DISPLAY_USAGE)
+      errStr := fmt.Sprintf("%s%s %s %s", "clinet.Activation.",funcName,"error:", "No activations found")
```

Comment:
The user should not be informed of the function name directly through an error message. Such information should be displayed in debug messages. Also, the error message is not very clear to someone that doesn't know the internal workings of the CLI.